SteamVR.cpp

```diff
@@ -156,6 +160,13 @@ bool GetSteamVRPositionalData(float *yaw, float *pitch, float *x, float *y, floa
 		//vr::VRSystem()->GetDeviceToAbsoluteTrackingPose(vr::TrackingUniverseSeated, 0.029, &trackedDevicePose, 1);
 		//vr::VRSystem()->GetDeviceToAbsoluteTrackingPose(vr::TrackingUniverseSeated, 0.042f, &g_hmdPose, 1);
 		//vr::VRSystem()->GetDeviceToAbsoluteTrackingPose(vr::TrackingUniverseSeated, 0.011f, &g_hmdPose, 1);
+		vr::Compositor_FrameTiming *frametiming = new vr::Compositor_FrameTiming();

```

Comment:
I don't see the `delete` for this `new`. I'll fix it. It's probably better to use a static var anyway, instead of allocating memory at every frame.

Reply:
You're completely right. My late night C++ skills are close to null 😁
I'll fix it later today.